dirty.rb

```diff
@@ -0,0 +1,8 @@
+a,b=2,1
```

Comment:
__[staging][rubocop]__
* Useless assignment to variable - `a`. Use `_` or `_a` as a variable name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Do not use parallel assignment. [:ref](https://github.com/bbatsov/ruby-style-guide#parallel-assignment)
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Useless assignment to variable - `b`. Use `_` or `_b` as a variable name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Surrounding space missing for operator `=`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)